Fomichev/HashingApp/HashingApp/MainForm.Designer.cs

```diff
@@ -0,0 +1,222 @@
+﻿namespace HashingApp
+{
+    partial class MainForm
+    {
+        /// <summary>
+        /// Required designer variable.
+        /// </summary>
+        private System.ComponentModel.IContainer components = null;
+
+        /// <summary>
+        /// Clean up any resources being used.
+        /// </summary>
+        /// <param name="disposing">true if managed resources should be disposed; otherwise, false.</param>
+        protected override void Dispose(bool disposing)
+        {
+            if (disposing && (components != null))
+            {
+                components.Dispose();
+            }
+            base.Dispose(disposing);
+        }
+
+        #region Windows Form Designer generated code
+
+        /// <summary>
+        /// Required method for Designer support - do not modify
+        /// the contents of this method with the code editor.
+        /// </summary>
+        private void InitializeComponent()
+        {
+            this.ProgramModeComboBox = new System.Windows.Forms.ComboBox();
+            this.HashAlgorithmComboBox = new System.Windows.Forms.ComboBox();
+            this.BlockSizeNumericUpDown = new System.Windows.Forms.NumericUpDown();
```

Comment:
I would use ComboBox for block size also, probably with some more readable names, like 
1 KB
2 KB
..
512 KB
1 MB

But your option is also ok